charts/incubator/etesync/values.yaml

```diff
@@ -0,0 +1,104 @@
+image:
+  repository: victorrds/etesync
+  pullPolicy: IfNotPresent
+  tag: 0.9.1@sha256:a880ed256dd78f14fb8dc9ab2e72ca2acebd3b8a87db65ea038f30abed27d896
+
+podSecurityContext:
+  runAsUser: 373
+  runAsGroup: 373
+  fsGroup: 373
+
+etesync:
+  ALLOWED_HOSTS: "*"
+  AUTO_SIGNUP: false
+  LANGUAGE_CODE: "en-us"
+  SUPER_USER: "admin"
+  SUPER_PASS: "TestPassword"
+  SUPER_EMAIL: "admin@example.com"
+  DEBUG: true
+  SHELL_DEBUG: true
+  DEBUG_DJANGO: true
```

Comment:
Maybe change those to false by default?
Also values on values.yaml and on questions.yaml should be in sync.
So the CI will run with the default values that a user will use if goes next next next.

Reply:
I actually had it like this in the beginning, but thought along the way it would be nice to see the debug output in the CI.

Reply:
> Also values on values.yaml and on questions.yaml should be in sync.

I re-synced values/questions with the exception of `ALLOWED_HOSTS`. I'd rather have the user set it to `*` consciously instead of defaulting to `*` which isn't recommended for production.